src/windows/input_helper.c

```diff
@@ -472,7 +472,7 @@ static int refresh_locale_list() {
 	// Get the number of layouts the user has activated.
 	int hkl_size = GetKeyboardLayoutList(0, NULL);
 	if (hkl_size > 0) {
-		logger(LOG_LEVEL_INFO,	"%s [%u]: GetKeyboardLayoutList(0, NULL) found %i layouts.\n",
+		logger(LOG_LEVEL_DEBUG,	"%s [%u]: GetKeyboardLayoutList(0, NULL) found %i layouts.\n",
```

Comment:
We should reduce the granularity of other events as well.  The keyboard and mouse stuff is far more annoying than this one.